src/bio/bio_monitor.c

```diff
@@ -735,16 +736,16 @@ auto_faulty_detect(struct bio_blobstore *bbs)
 	if (bbs->bb_state == BIO_BS_STATE_FAULTY)
```

Comment:
I don't think you need this check now you have the one below (`!NORMAL && !OUT`) so this can be removed.

Reply:
Indeed, this could be removed now. I'll keep it in mind and cleanup it later.